packages/plugin-core/src/test/suite-integ/LookupUtils.test.ts

```diff
@@ -0,0 +1,210 @@
+import { DVault } from "@dendronhq/common-all";
+import { NoteTestUtilsV4 } from "@dendronhq/common-test-utils";
+import { TestEngineUtils } from "@dendronhq/engine-test-utils";
+import { describe } from "mocha";
+import * as vscode from "vscode";
+import {
+  CONTEXT_DETAIL,
+  FULL_MATCH_DETAIL,
+  HIERARCHY_MATCH_DETAIL,
+  PickerUtilsV2,
+} from "../../components/lookup/utils";
+import { TIMEOUT } from "../testUtils";
+import { expect } from "../testUtilsv2";
+import { runLegacyMultiWorkspaceTest, setupBeforeAfter } from "../testUtilsV3";
+
+function setupNotesForTest({
+  wsRoot,
+  vaults,
+  vault1,
+  vault2,
+  vault3,
+}: {
+  wsRoot: string;
+  vaults: DVault[];
+  vault1?: string[];
+  vault2?: string[];
+  vault3?: string[];
+}) {
+  if (vault1) {
+    vault1.forEach(async (str) => {
+      await NoteTestUtilsV4.createNote({
+        vault: TestEngineUtils.vault1(vaults),
+        wsRoot,
+        fname: str,
+        body: "",
+        genRandomId: true,
+      });
+    });
+  }
+
+  if (vault2) {
+    vault2.forEach(async (str) => {
+      await NoteTestUtilsV4.createNote({
+        vault: TestEngineUtils.vault2(vaults),
+        wsRoot,
+        fname: str,
+        body: "",
+        genRandomId: true,
+      });
+    });
+  }
+
+  if (vault3) {
+    vault3.forEach(async (str) => {
+      await NoteTestUtilsV4.createNote({
+        vault: TestEngineUtils.vault3(vaults),
+        wsRoot,
+        fname: str,
+        body: "",
+        genRandomId: true,
+      });
+    });
+  }
+}
+
+/**
+ * Tests the Vault Recommendation For New Notes Functionality in Utils
+ */
+suite.only("Lookup Utils Test", function runSuite() {
```

Comment:
remove `.only`. also, feel free to add it to the [husky](https://github.com/dendronhq/dendron/blob/dev/hooks/pre-push.js#L20:L20) scripit

Reply:
good catch :) let me add

Reply:
will add husky change as part of a separate review.